snuba/settings.py

```diff
@@ -116,6 +116,9 @@
     },
 }  # (dataset name: (referrer: percentage))
 
+COLUMN_SPLIT_MAX_LIMIT = 1000
+COLUMN_SPLIT_MAX_RESULTS = 5000
```

Comment:
Any basis for these numbers or are they just approximations?

Reply:
The limit is the default query limit, which is the maximum sentry uses (other than csv exporter which is causing the issue).
Max result is 32 bytes (the string event it converted into bytes we send to clickhouse) times 5000 ->  160kb. I don't know if clickhouse measures that based on 8 bit characters or unicode, so I gave it twice as much. + Margin for the rest of the query to be comfortably under the 500kb limit.

Will check how clickhouse measures later.